docs/changelog.rst

```diff
@@ -36,7 +36,121 @@ We use a one-line description of every pull request.
 ..
 .. - [#XXX] TITLE (AUTHOR)
 
-.. _pyirf_0p3_release:
+
+.. _pyirf_0p3p0_release:
+
+`0.3.0 <https://github.com/cta-observatory/pyirf/releases/tag/v0.3.0>`__ (2020-10-05)
+-------------------------------------------------------------------------------------
+
+Summary
++++++++
+
+- Released October 5th, 2020
+- 5 Contributors
+
+Contributors
+++++++++++++
+
+In order of number of commits:
+
+- Maximilian Nöthe
+- Michele Peresano
+- Noah Biederbeck
+- Lukas Nickel
+- Gaia Verna
+
+
+Description
++++++++++++
+
+This release is the result of the IRF sprint week in September 2020.
+
+Many bug fixes and improvements were made to the code.
+In the central and upper energy range, ``pyirf`` now reproduces the ``EventDisplay`` sensitivity
+exactly, the lower energy bins still show some disagreement.
+
+The cut optimization seems not yet to be the same as EventDisplay's and will be further investigated.
+
```

Comment:
I think we should be a bit more specific about the context to avoid misunderstandings: we now reproduce those results because we have a specific example that tries to follow that specific pipeline approach.

Reply:
Yes, agreed.